api/types/transfer/imagestore.proto

```diff
@@ -80,3 +84,19 @@ message ImageReference {
 	// Only used if IsPrefix is true.
 	bool skip_named_digest = 5;
 }
+
+// ContentPolicy is used to determine how content related
+// the the image should be retained
+enum ContentPolicy {
```

Comment:
"policy" sounds like a security policy.

Maybe:
```suggestion
enum MetadataFilter {
```

Reply:
It seems like `ContentFilter` could also work, depending on your point of reference (are you filtering to keep something or discard something?)